common/crypto/dkg/src/bte/mod.rs

```diff
@@ -243,7 +243,7 @@ impl Zeroize for Tau {
     }
 }
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd)]
+#[derive(Copy, Clone, Debug, Default, PartialEq, Eq, PartialOrd)]
 pub struct Epoch(EpochStore);
```

Comment:
I think we might be actually possible to completely get rid of `Epoch` since we don't have to do any key updating

Reply:
See answer in main thread i.e. I would leave the epoch implementation there, just in case